routes/cube_routes.js

```diff
@@ -1556,6 +1556,119 @@ router.get('/download/plaintext/:id', async (req, res) => {
   }
 });
 
+function shuffle(a) {
+  for (let i = a.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [a[i], a[j]] = [a[j], a[i]];
+  }
+  return a;
+}
+
+router.post('/startsealed/:id', async (req, res) => {
+  try {
+    const user = await User.findById(req.user);
+
+    if (!user) {
+      req.flash('danger', 'Please Login to build a sealed deck.');
+      return res.redirect(`cube/playtest/${req.params.id}`);
+    }
+    const params = {
+      packs: parseInt(req.body.packs, 10),
+      cards: parseInt(req.body.cards, 10),
+    };
+
+    const numCards = params.packs * params.cards;
+
+    const cube = await Cube.findOne(buildIdQuery(req.params.id), '_id name draft_formats card_count type cards owner');
+
+    if (!cube) {
+      req.flash('danger', 'Cube not found');
+      return res.status(404).render('misc/404', {});
+    }
+
+    if (cube.cards.length < numCards) {
+      throw new Error('Could not create sealed pool: not enough cards');
+    }
+
+    const source = shuffle(cube.cards).slice(0, numCards);
```

Comment:
Why can't you use the pack generation code we use for drafts?

Reply:
This is just a basic implementation that can be extended later